src/analytics/Properties.tsx

```diff
@@ -1381,6 +1381,13 @@ interface DappShortcutsProperties {
   [DappShortcutsEvents.dapp_shortcuts_reward_tx_rejected]: DappShortcutClaimRewardEvent
 }
 
+interface TransactionDetailsProperties {
+  [TransactionDetailsEvents.transaction_details_tap_details]: undefined
```

Comment:
should we add the transaction type and status to all these events? so we can learn if the tx details screen is most helpful for certain transaction types or statuses :)

Reply:
Sure, that's a great idea!